torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -48,6 +48,19 @@ std::vector<int64_t> TileAssignmentDimensions(
 
 }  // namespace
 
+bool ShouldUseVirtualDevice() {
+  bool use_virtual_device = xla::sys_util::GetEnvBool("XLA_USE_SPMD", false);
```

Comment:
nit, `static bool`. User should not be able to change env var after process is started

Reply:
oh nvm, saw the below function.